ftp/1.6/modules/ROOT/pages/ftp-studio-configuration.adoc

```diff
@@ -0,0 +1,124 @@
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:ftp-documentation.adoc[FTP Connector Reference Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add FTP Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `ftp` in the search field.
+. Click *FTP Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with SFTP Connector:
+
+* *On New or Updated File* +
+Initiates a flow by polling a directory for files that have been created or updated and generates a message for each file found.
+* *HTTP Listener* +
+Initiates a flow each time it receives a request on the configured host and port.
+* *Scheduler* +
+Initiates a flow when a time-based condition is met.
+
+For example, to configure the On New or Updated File source, follow these steps:
+
+. In the *Mule Palette* view, select *On New or Updated File*.
+. Drag *On New or Updated File* to the Studio canvas.
+. On the *On New or Updated File* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of *On New or Updated File* in the app.
+. On the *General* tab, specify the connection information for the connector.
+. Set *Host* to specify the FTP server host, for example, `localhost` or `192.168.0.1`.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. On the *On New or Updated File* configuration screen, specify a *Scheduling Strategy* value and optionally complete other fields.
```

Comment:
The part about optional fields was a little confusing. Maybe make it its own sentence? You could end the first sentence after "value." Then, the second sentence could be:
"The remaining fields are optional to complete."